eng/targets/Helix.Common.props

```diff
@@ -8,57 +8,57 @@
     <HelixQueueArmDebian11>(Debian.11.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm64v8-20211001171229-97d8652</HelixQueueArmDebian11>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true'">
+  <!-- Don't use <Choose/> for $(IsWindowsOnlyTest) checks. <When/> Condition is evaluated before project load. -->
+  <ItemGroup Condition=" '$(IsWindowsOnlyTest)' != 'true' ">
     <HelixAvailablePlatform Include="Windows" />
     <HelixAvailablePlatform Include="OSX" />
     <HelixAvailablePlatform Include="Linux" />
   </ItemGroup>
-
-  <ItemGroup Condition="'$(IsWindowsOnlyTest)' == 'true'">
+  <ItemGroup Condition=" '$(IsWindowsOnlyTest)' == 'true' ">
     <HelixAvailablePlatform Include="Windows" />
   </ItemGroup>
 
-  <!-- x64 PR(ci.yaml) required queues for internal and public cases -->
-  <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' != 'true'">
-    <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Windows.11.Amd64.ClientPre.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="OSX" />
-  </ItemGroup>
-
-  <!-- x64 Queues for public helix-matrix.yml and quarantine pipelines, except in windows-only cases -->
-  <ItemGroup Condition="'$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true' AND '$(_UseHelixOpenQueues)' == 'true' AND '$(IsWindowsOnlyTest)' != 'true'">
-    <!-- Linux -->
-    <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Ubuntu.2004.Amd64.Open" Platform="Linux" />
-
-    <!-- Mac -->
-    <HelixAvailableTargetQueue Include="OSX.1015.Amd64.Open" Platform="OSX" />
-
-    <!-- Containers -->
-    <HelixAvailableTargetQueue Include="$(HelixQueueAlpine314)" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="$(HelixQueueDebian11)" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="$(HelixQueueFedora34)" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="$(HelixQueueMariner)" Platform="Linux" />
-  </ItemGroup>
+  <!--
+    Usually do not need to check $(_UseHelixOpenQueues), $(RunQuarantinedTests) or $(IsWindowsOnlyTest).
+    $(_UseHelixOpenQueues) handling in helix.proj and Helix.targets is sufficient. We have no quarantined queues
+    at the moment. And, most Windows-specific checks are handled using @(HelixAvailablePlatform) in Helix.props.
+  -->
+  <Choose>
+    <When Condition=" '$(IsHelixDaily)' != 'true' ">
```

Comment:
We should take this opportunity to rename IsHelixDaily to something more correct and clear to make this more readable

Reply:
Maybe flip it and make this IsPRCheck to be explicit

Reply:
done